### ISSUE TITLE
Finding mentions are very difficult in busy rooms

### ISSUE THREAD

**Issue by Sleuth56** [vector-im/element-web#19735]
#### What would you like to do?
Find messages I was mentioned in.

#### Why would you like to do it?
Finding messages that mentioned me is next to impossible in large rooms.

#### How would you like to achieve it?
Make the green button to skip to last read change to red when you have a mention. Clicking this button will bring you to that message first. Then the button will change back to green.

![Screenshot_20211112_131315](https://user-images.githubusercontent.com/34520077/141515238-da65cf01-c178-4226-9d98-9d76f583e83d.png)

**Comment by frandavid100** [comment]
I love this.

**Comment by t3chguy** [comment]
There is also the bell icon top right which shows all of your mentions and lets you jump to them

**Comment by Palid** [comment]
Agree with the general idea, in Telegram for instance the up arrow jumps to the first mention if you have those. I'm not sure if it works like that in Element, as it never did for me. 🥲

**Comment by Sleuth56** [comment]
@Palid No, Element does not work that way. Element brings you back to where your last read receipt is.
